components/service-operator/internal/aws/sqs.go

```diff
@@ -0,0 +1,43 @@
+package aws
+
+import (
+	queue "github.com/alphagov/gsp/components/service-operator/apis/queue/v1beta1"
+
+	awscloudformation "github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/awslabs/goformation/cloudformation"
+	"github.com/awslabs/goformation/cloudformation/resources"
+)
+
+const (
+	SQSResourceName = "SQSQueue"
+
+	SQSOutputURL = "QueueURL"
+)
+
+type SQS struct {
+	SQSConfig *queue.SQS
+}
+
+func (s *SQS) Template(stackName string, tags []resources.Tag) *cloudformation.Template {
+	template := cloudformation.NewTemplate()
+
+	template.Resources[SQSResourceName] = &resources.AWSSQSQueue{
+		QueueName: s.SQSConfig.Spec.AWS.QueueName,
```

Comment:
This may have been raised elsewhere already...but I don't think the queue name should be a property of the spec. The queue name should be the metadata.name of the sqsqueue resource.